CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ This CHANGELOG follows the format listed at [Our CHANGELOG Guidelines ](https://
 Which is based on [Keep A Changelog](http://keepachangelog.com/)
 
 ## [Unreleased]
+### Changed - 2017-12-11
```

Comment:
We only date the release not the contribution (we have that in git history). If you want to read more about our [changelog  conventions](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md). This line should look like this:
```
### Changed
```